site/docs/v1/tech/tutorials/integrate-angular.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: doc
+title: Integrate Your Angular Application With FusionAuth
+description: Integrate your Angular application with FusionAuth
+navcategory: getting-started
+prerequisites: nodejs
+technology: Angular
+language: TypeScript
+---
+
+{% include docs/integration/_intro.md %}
+
+## Prerequisites
+
+{% include docs/integration/_prerequisites.md %}
+
+## Download and Install FusionAuth
+
+{% include docs/integration/_install-fusionauth.md %}
+
+## Create a User and an API Key
+
+{% include docs/integration/_add-user.md language=page.language %}
+
+## Configure FusionAuth
+
+Next, you need to set up FusionAuth. This can be done in different ways, but we are going to use the {{page.language}} client library. The instructions below use `npm` on the command line, but you can use the client library with an IDE of your preference as well.
+
+First, make a directory:
+
+```shell
+mkdir setup-fusionauth && cd setup-fusionauth
+```
+
+Now, copy and paste the following code into `package.json`.
+
+```javascript
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-client-libraries/main/typescript/package.json %}
+```
+
+Now you need to install the dependencies in `package.json`.
+
+```shell
+npm install
+```
+
+Then copy and paste the following code into `setup.js`. This file uses the [FusionAuth API](/docs/v1/tech/apis/) to configure an Application, CORS, and more to allow for easy integration.
+
+```javascript
+{% remote_include https://raw.githubusercontent.com/sonderformat-llc/fusionauth-example-client-libraries/angular/typescript/setup-angular.js %}
+```
+
+Then, you can run the setup script.
+
+```shell
+fusionauth_api_key=YOUR_API_KEY_FROM_ABOVE npm run setup
```

Comment:
```suggestion
fusionauth_api_key=YOUR_API_KEY_FROM_ABOVE node setup.js
```

Reply:
We renamed the setup scripts sometime ago to accommodate different files, so you could either:

1) Change any mentions to `npm run setup` and `setup.js` to `npm run setup-angular` and `setup-angular.js`; or
2) Change `npm run setup` to `node setup.js`